### PR TITLE
semantics ,forms, attributes in HTML

### PR DESCRIPTION
**Pull request by ioandodita069**

### CODE REVIEW

File: html/attributes.html [hunk 1]
```diff
@@ -1,26 +1,31 @@
...
+    <h1>The form autocomplete attribute</h1>
+
+    <p>Fill in and submit the form, then reload the page, start to fill in the form again - and see how autocomplete
+        works.</p>
 
-<p>Fill in and submit the form, then reload the page, start to fill in the form again and see how autocomplete works.</p>
+    <p>Then, try to set autocomplete to "off".</p>
 
-<p>Then, try to set autocomplete to "off".</p>
+    <form action="/action_page.php" autocomplete="on">
```
Comment (pancudaniel7) [inline html/attributes.html:12]: Question:
- Did you understood what dose this line of html ?

File: html/attributes.html [hunk 2]
```diff
@@ -1,26 +1,31 @@
...
-</form>
-<h1>The form novalidate attribute</h1>
+    <p>The novalidate attribute indicates that the form input is not to be validated on submit:</p>
 
-<form action="/action_page.php" novalidate>
-  <label for="email">Enter your email:</label>
-  <input type="email" id="email" name="email"><br><br>
-  <input type="submit">
-</form>
+    <form action="/action_page.php" novalidate>
```
Comment (pancudaniel7) [inline html/attributes.html:23]: Or this one ?

File: html/attributes.html (hunk 1 above)
Comment (ioandodita069) [reply html/attributes.html:12]: yes,I did

File: html/attributes.html (hunk 2 above)
Comment (ioandodita069) [reply html/attributes.html:23]: this one too

**Pull request merged**